src/Parallel/DistributedObject.hpp

```diff
@@ -1185,9 +1185,14 @@ void DistributedObject<ParallelComponent,
         "an issue.");
     sys::abort("");
   }
-  const std::string message = MakeString{}
-                              << "Message: " << exception.what() << "\nType: "
-                              << pretty_type::get_runtime_type_name(exception);
+  const std::string message =
+      MakeString{} << "Component: "
+                   << pretty_type::get_name<parallel_component>() << "\n"
```

Comment:
`pretty_type::name<parallel_component>()` or else all control system (or interpolation target) singletons will have the same name.